src/main/java/software/amazon/awssdk/eventstreamrpc/ServiceOperationMappingContinuationHandler.java

```diff
@@ -101,57 +112,63 @@ protected void onConnectRequest(List<Header> headers, byte[] payload) {
                 LOGGER.trace(String.format("%s running authentication handler", serviceHandler.getServiceName()));
                 authenticationData = authentication.apply(headers, payload);
                 if (authenticationData == null) {
-                    throw new IllegalStateException(String.format("%s authentication handler returned null", serviceHandler.getServiceName()));
+                    throw new IllegalStateException(
+                            String.format("%s authentication handler returned null", serviceHandler.getServiceName()));
                 }
-                LOGGER.info(String.format("%s authenticated identity: %s", serviceHandler.getServiceName(), authenticationData.getIdentityLabel()));
+                LOGGER.info(String.format("%s authenticated identity: %s", serviceHandler.getServiceName(),
+                        authenticationData.getIdentityLabel()));
 
                 final Authorization authorizationDecision = authorization.apply(authenticationData);
                 switch (authorizationDecision) {
-                    case ACCEPT:
+                    case ACCEPT :
                         LOGGER.info("Connection accepted for " + authenticationData.getIdentityLabel());
                         responseMessageFlag[0] = MessageFlags.ConnectionAccepted.getByteValue();
                         break;
-                    case REJECT:
+                    case REJECT :
                         LOGGER.info("Connection rejected for: " + authenticationData.getIdentityLabel());
                         break;
-                    default:
-                        //got a big problem if this is the outcome. Someone forgot to update this switch-case
-                        throw new RuntimeException("Unknown authorization decision for " + authenticationData.getIdentityLabel());
+                    default :
+                        // got a big problem if this is the outcome. Someone forgot to update this
+                        // switch-case
+                        throw new RuntimeException(
+                                "Unknown authorization decision for " + authenticationData.getIdentityLabel());
                 }
-            } else { //version mismatch
+            } else { // version mismatch
                 LOGGER.warn(String.format("Client version {%s} mismatches server version {%s}",
                         versionHeader.isPresent() ? versionHeader.get() : "null",
                         Version.getInstance().getVersionString()));
             }
         } catch (Exception e) {
-            LOGGER.error(String.format("%s occurred while attempting to authN/authZ connect: %s", e.getClass(), e.getMessage()), e);
+            LOGGER.error(String.format("%s occurred while attempting to authN/authZ connect: %s", e.getClass(),
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential log injection detected. Ensure all untrusted input is properly sanitized before logging. Use parameterized logging or validate input against a allow list to prevent log injection vulnerabilities. Consider using a dedicated logging library&#39;s built-in sanitization features when available. Learn more - https://cwe.mitre.org/data/definitions/117.html